integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/PatientFhirResourceProviderIntegrationTest.java

```diff
@@ -659,7 +745,22 @@ public void shouldReturnForPatientEverythingWhenCountIsSpecifiedAsXml() throws E
 		validResources.add(ENCOUNTER_UUID_3);
 		validResources.add(MEDICATION_REQUEST_UUID_1);
 		validResources.add(MEDICATION_REQUEST_UUID_2);
-
+		
 		return validResources;
 	}
+
+	private Set<ResourceType> getEverythingValidResourceTypes() {
+		Set<ResourceType> validTypes = new HashSet<>();
+
+		validTypes.add(ResourceType.Patient);
+		validTypes.add(ResourceType.Observation);
+		validTypes.add(ResourceType.Procedure);
```

Comment:
We don't use `Procedure` in the _revinclude implementation for Patient resource.